diff --git a/meta-leda-bsp/conf/machine/qemux86-64-extra.conf b/meta-leda-bsp/conf/machine/qemux86-64-extra.conf
index 193355c..faafc9f 100644
--- a/meta-leda-bsp/conf/machine/qemux86-64-extra.conf
+++ b/meta-leda-bsp/conf/machine/qemux86-64-extra.conf
@@ -10,3 +10,7 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += "v86d"
 MACHINE_EXTRA_RRECOMMENDS = "kernel-module-snd-ens1370 kernel-module-snd-rawmidi"
 WKS_FILE ?= "qemux86-grub-efi.wks"
 do_image_wic[depends] += "syslinux:do_populate_sysroot syslinux-native:do_populate_sysroot mtools-native:do_populate_sysroot dosfstools-native:do_populate_sysroot"
+
+MACHINE_FEATURES:append = " pcbios efi"
+EXTRA_IMAGEDEPENDS += "ovmf"
+PREFERRED_RPROVIDER_virtual-grub-bootconf = "rauc-qemu-grubconf"
diff --git a/meta-leda-distro/conf/distro/include/leda-qemu-settings.inc b/meta-leda-distro/conf/distro/include/leda-qemu-settings.inc
index cd20427..c614337 100644
--- a/meta-leda-distro/conf/distro/include/leda-qemu-settings.inc
+++ b/meta-leda-distro/conf/distro/include/leda-qemu-settings.inc
@@ -16,7 +16,8 @@ QB_MEM = "-m 4096"
 QB_MEM:qemuarm = "-m 2048"
 
 # No graphical user interface
-QB_OPT_APPEND:append = " -serial mon:stdio -serial null -nographic"
+# QB_OPT_APPEND:append = " -serial mon:stdio -serial null"
+QB_OPT_APPEND:append = " -nographic"
 
 # Emulate a CAN-Bus device
 QB_OPT_APPEND:append = " -object can-bus,id=canbus0 "
diff --git a/meta-leda-distro/conf/distro/include/leda-selfupdate-rauc.inc b/meta-leda-distro/conf/distro/include/leda-selfupdate-rauc.inc
index 1a1f057..d391729 100644
--- a/meta-leda-distro/conf/distro/include/leda-selfupdate-rauc.inc
+++ b/meta-leda-distro/conf/distro/include/leda-selfupdate-rauc.inc
@@ -11,7 +11,8 @@
 # * SPDX-License-Identifier: Apache-2.0
 # ********************************************************************************/
 
+# Machine specific, see meta-leda/meta-leda-bsp/conf/machine/qemux86-64-extra.conf
 # Enable GRUB Bootloader, so that we can chose which partition to boot.
-MACHINE_FEATURES:append = " pcbios efi"
-EXTRA_IMAGEDEPENDS += "ovmf"
-PREFERRED_RPROVIDER_virtual-grub-bootconf = "rauc-qemu-grubconf"
+# MACHINE_FEATURES:append = " pcbios efi"
+# EXTRA_IMAGEDEPENDS += "ovmf"
+# PREFERRED_RPROVIDER_virtual-grub-bootconf = "rauc-qemu-grubconf"
diff --git a/meta-leda-distro/recipes-sdv-distro/base-files/qemuarm/fstab b/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/base-files/qemuarm/fstab
rename to meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab
diff --git a/meta-leda-distro/recipes-sdv-distro/base-files/qemuarm64/fstab b/meta-leda-distro/recipes-core/base-files/base-files/qemuarm64/fstab
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/base-files/qemuarm64/fstab
rename to meta-leda-distro/recipes-core/base-files/base-files/qemuarm64/fstab
diff --git a/meta-leda-distro/recipes-sdv-distro/base-files/qemux86-64/fstab b/meta-leda-distro/recipes-core/base-files/base-files/qemux86-64/fstab
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/base-files/qemux86-64/fstab
rename to meta-leda-distro/recipes-core/base-files/base-files/qemux86-64/fstab
diff --git a/meta-leda-distro/recipes-sdv-distro/base-files/raspberrypi4-64/fstab b/meta-leda-distro/recipes-core/base-files/base-files/raspberrypi4-64/fstab
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/base-files/raspberrypi4-64/fstab
rename to meta-leda-distro/recipes-core/base-files/base-files/raspberrypi4-64/fstab
diff --git a/meta-leda-distro/recipes-sdv-distro/base-files_%.bbappend b/meta-leda-distro/recipes-core/base-files/base-files_%.bbappend
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/base-files_%.bbappend
rename to meta-leda-distro/recipes-core/base-files/base-files_%.bbappend
diff --git a/meta-leda-distro/recipes-sdv-distro/files/systemd-conf-files/can0.network b/meta-leda-distro/recipes-core/systemd/systemd-conf/can0.network
similarity index 100%
rename from meta-leda-distro/recipes-sdv-distro/files/systemd-conf-files/can0.network
rename to meta-leda-distro/recipes-core/systemd/systemd-conf/can0.network
diff --git a/meta-leda-distro/recipes-sdv-distro/systemd-conf_%.bbappend b/meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend
similarity index 79%
rename from meta-leda-distro/recipes-sdv-distro/systemd-conf_%.bbappend
rename to meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend
index 3929e6f..5de3ad2 100644
--- a/meta-leda-distro/recipes-sdv-distro/systemd-conf_%.bbappend
+++ b/meta-leda-distro/recipes-core/systemd/systemd-conf_%.bbappend
@@ -13,13 +13,13 @@
 #
 # Enable CAN bus network by default
 #
-FILESEXTRAPATHS:prepend := "${THISDIR}/systemd-conf-files:"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 
 SRC_URI += " file://can0.network"
 
 FILES:${PN} += " ${sysconfdir}/systemd/network/can0.network"
 
 do_install:append() {
-    install -d ${D}${sysconfdir}/systemd/network
-    install -m 0644 ${WORKDIR}/can0.network ${D}${sysconfdir}/systemd/network
-}
\ No newline at end of file
+    install -D -m 0644 ${WORKDIR}/can0.network ${D}${systemd_unitdir}/network
+}
