diff --git a/meta-leda-bsp/conf/machine/qemuarm-extra.conf b/meta-leda-bsp/conf/machine/qemuarm-extra.conf
index ad6c901..d5bec7d 100644
--- a/meta-leda-bsp/conf/machine/qemuarm-extra.conf
+++ b/meta-leda-bsp/conf/machine/qemuarm-extra.conf
@@ -6,4 +6,5 @@ require conf/machine/common-qemu-arm.inc
 
 WKS_FILES = "qemuarm.wks"
 UBOOT_MACHINE = "qemu_arm_defconfig"
-KERNEL_IMAGETYPE = "uImage"
\ No newline at end of file
+KERNEL_IMAGETYPE = "zImage"
+KERNEL_BOOTCMD = "bootz"
diff --git a/meta-leda-bsp/recipes-bsp/uboot/rauc_%.bbappend b/meta-leda-bsp/recipes-bsp/uboot/rauc_%.bbappend
index 5b31f82..02cbfdb 100644
--- a/meta-leda-bsp/recipes-bsp/uboot/rauc_%.bbappend
+++ b/meta-leda-bsp/recipes-bsp/uboot/rauc_%.bbappend
@@ -13,4 +13,5 @@
 
 
 RDEPENDS:${PN}:qemuarm64 += "u-boot-fw-utils u-boot-env libubootenv"
+RDEPENDS:${PN}:qemuarm += "u-boot-fw-utils u-boot-env libubootenv"
 
diff --git a/meta-leda-bsp/recipes-bsp/uboot/u-boot_%.bbappend b/meta-leda-bsp/recipes-bsp/uboot/u-boot_%.bbappend
index ae224dd..3237e6f 100644
--- a/meta-leda-bsp/recipes-bsp/uboot/u-boot_%.bbappend
+++ b/meta-leda-bsp/recipes-bsp/uboot/u-boot_%.bbappend
@@ -16,4 +16,5 @@ SUMMARY = "U-boot boot scripts for targets"
 LICENSE = "Apache-2.0"
 
 inherit rauc-integration
-include uboot-targets/qemuarm64.inc
\ No newline at end of file
+include uboot-targets/qemuarm64.inc
+include uboot-targets/qemuarm.inc
\ No newline at end of file
diff --git a/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab b/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab
index b1e923a..14126d9 100644
--- a/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab
+++ b/meta-leda-distro/recipes-core/base-files/base-files/qemuarm/fstab
@@ -7,6 +7,7 @@ tmpfs                /run                 tmpfs      mode=0755,nodev,nosuid,stri
 tmpfs                /var/volatile        tmpfs      defaults              0  0
 
 # Add mount for boot, grubenv and data partition
-LABEL=boot           /boot                auto       defaults,nofail,noatime  0  0
+LABEL=BOOT           /boot                vfat       defaults,nofail,noatime  0  0
 LABEL=grubenv        /grubenv             auto       defaults,nofail,noatime  0  0
 LABEL=data           /data                auto       defaults,nofail,noatime  0  2
+
diff --git a/meta-leda-distro/wic/qemuarm.cfg b/meta-leda-distro/wic/qemuarm.cfg
deleted file mode 100644
index 8591034..0000000
--- a/meta-leda-distro/wic/qemuarm.cfg
+++ /dev/null
@@ -1,36 +0,0 @@
-# /********************************************************************************
-# * Copyright (c) 2022 Contributors to the Eclipse Foundation
-# *
-# * See the NOTICE file(s) distributed with this work for additional
-# * information regarding copyright ownership.
-# *
-# * This program and the accompanying materials are made available under the
-# * terms of the Apache License 2.0 which is available at
-# * https://www.apache.org/licenses/LICENSE-2.0
-# *
-# * SPDX-License-Identifier: Apache-2.0
-# ********************************************************************************/
-
-menu title Eclipse Leda (QEMU ARM)
-
-timeout 50
-
-default sdv-full
-
-# vda2 = rescue
-# vda3 = full
-# vda4 = minimal
-# vda5 = data
-
-label sdv-full
-    menu label SDV Full (with tools)
-    kernel /uImage
-    append root=/dev/vda4 console=ttyAMA0,115200n8 panic=5 ip=dhcp ip=192.168.7.2::192.168.7.1:255.255.255.0 rootwait rauc.slot=SDV_A
-label sdv-minimal
-    menu label SDV Minimal
-    kernel /uImage
-    append root=/dev/vda5 console=ttyAMA0,115200n8 panic=5 ip=dhcp ip=192.168.7.2::192.168.7.1:255.255.255.0 rootwait rauc.slot=SDV_B
-label sdv-rescue
-    menu label SDV Rescue
-    kernel /uImage
-    append root=/dev/vda3 console=ttyAMA0,115200n8 panic=5 ip=dhcp ip=192.168.7.2::192.168.7.1:255.255.255.0 rootwait rauc.external
diff --git a/meta-leda-distro/wic/qemuarm.wks b/meta-leda-distro/wic/qemuarm.wks
index fcb5c8a..6ff2ede 100644
--- a/meta-leda-distro/wic/qemuarm.wks
+++ b/meta-leda-distro/wic/qemuarm.wks
@@ -21,10 +21,10 @@
 # * a common data partition
 
 # Boot Loader Configuration, e.g. boot menu
-bootloader --ptable gpt --configfile="qemuarm.cfg"
+bootloader --ptable gpt
 
 # Boot partition (/dev/vda1)
-part --source bootimg-partition --ondisk vda --fstype=vfat --label boot --active --align 4 --size 16 --sourceparams="loader=u-boot" --use-uuid
+part --source bootimg-partition --ondisk vda --fstype=vfat --label BOOT --active --align 4096 --size 100 --use-uuid
 
 # Second Bootloader Partition (empty, for future updates)
 part --fixed-size 10M --ondisk vda --align 4096 --no-table
diff --git a/meta-leda-distro/wic/qemuarm64.wks b/meta-leda-distro/wic/qemuarm64.wks
index d2da4f7..7af1401 100644
--- a/meta-leda-distro/wic/qemuarm64.wks
+++ b/meta-leda-distro/wic/qemuarm64.wks
@@ -35,4 +35,4 @@ part --fixed-size 10M --ondisk vda --align 4096
 part --source rootfs --rootfs-dir=sdv-image-rescue --ondisk vda --fstype=ext4 --label rescue --align 1024
 part --source rootfs --rootfs-dir=sdv-image-full --ondisk vda --fstype=ext4 --label root_a --align 4096
 part --source rootfs --rootfs-dir=sdv-image-minimal  --ondisk vda --fstype=ext4 --label root_b --align 4096
-part --fixed-size 2048M --source rootfs --rootfs-dir=sdv-image-data --ondisk vda --fstype=ext4 --label data --align 4096
+part --fixed-size 2048M --source rootfs --rootfs-dir=sdv-image-data --ondisk vda --fstype=ext4 --label data --align 4096
\ No newline at end of file
